emcc.py

```diff
@@ -28,13 +28,18 @@
 if __name__ == '__main__':
   ToolchainProfiler.record_process_start()
 
-import os, sys, shutil, tempfile, subprocess, shlex, time, re, logging, urllib
+import os, sys, shutil, tempfile, subprocess, shlex, time, re, logging
 from subprocess import PIPE
 from tools import shared, jsrun, system_libs
 from tools.shared import execute, suffix, unsuffixed, unsuffixed_basename, WINDOWS, safe_move
 from tools.response_file import read_response_file
 import tools.line_endings
 
+try:
```

Comment:
maybe here and in the other places add a comment saying this is for 2/3 support? that way when we can remove 2 support we could search for such things and remove the 2 parts. also it looks weird by itself, not sure it's obvious it's for 2/3 compat.

Reply:
That will be great, done.

(Will we be able to remove Py2 support in 2025? I hope so.)